Windows/InputBox.cpp

```diff
@@ -108,7 +108,7 @@ bool InputBox_GetWString(HINSTANCE hInst, HWND hParent, const wchar_t *title, co
 		return false;
 }
 
-bool InputBox_GetHex(HINSTANCE hInst, HWND hParent, const wchar_t *title, u32 defaultvalue, u32 &outvalue)
+bool InputBox_GetBin(HINSTANCE hInst, HWND hParent, const wchar_t *title, u32 defaultvalue, u32 &outvalue)
```

Comment:
How does this difer from InputBox_GetHex?  Do we need two?

-[Unknown]

Reply:
ok, it's gone